### PR TITLE
Native -ios - Env variable groups

### PR DESCRIPTION
**Pull request by K-Codemagic**

### CODE REVIEW

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 1]
```diff
@@ -140,6 +142,6 @@ workflows:
       - $HOME/Library/Developer/Xcode/DerivedData/**/Build/**/*.dSYM
     publishing:
       app_store_connect:
-        apple_id: your_apple_id@example.com  # PUT YOUR APPLE ID HERE
-        password: Encrypted(...) # PUT YOUR APP-SPECIFIC-PASSWORD HERE https://support.apple.com/en-us/HT204397
+        apple_id: $APPLE_ID
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md:145]: use new publishing method

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 2]
```diff
@@ -140,6 +142,6 @@ workflows:
...
+        password: $APP_SPECIFIC_PASSWORD
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md:146]: use new publishing method
```suggestion
                    app_store_connect:   # https://docs.codemagic.io/publishing-yaml/distribution              
              api_key: $APP_STORE_CONNECT_PRIVATE_KEY         # Contents of the API key, can also reference environment variable such as $APP_STORE_CONNECT_PRIVATE_KEY
              key_id: $APP_STORE_CONNECT_KEY_IDENTIFIER       # Alphanumeric value that identifies the API key, can also reference environment variable such as $APP_STORE_CONNECT_KEY_IDENTIFIER
              issuer_id: $APP_STORE_CONNECT_ISSUER_ID         # Alphanumeric value that identifies who created the API key, can also reference environment variable such as $APP_STORE_CONNECT_ISSUER_ID
              submit_to_testflight: false                     # Optional boolean, defaults to false. Whether or not to submit the uploaded build to TestFlight beta review. Required for distributing to beta groups. Note: This action is performed during post-processing.
            # beta_groups:                                  # Specify the names of beta tester groups that will get access to the build once it has passed beta review. 
            #       - group name 1
            #       - group name 2
```

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 3]
```diff
@@ -99,12 +99,14 @@ workflows:
   ios-workflow:
     name: ios_workflow
     environment:
+      groups:
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md:102]: add env group for ios credentials for publishing

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

File: content/yaml-quick-start/building-a-native-ios-app.md (hunk 2 above)
Comment (K-Codemagic) [reply content/yaml-quick-start/building-a-native-ios-app.md]: Changes have been made

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 4]
```diff
@@ -140,6 +138,12 @@ workflows:
       - $HOME/Library/Developer/Xcode/DerivedData/**/Build/**/*.dSYM
     publishing:
       app_store_connect:
-        apple_id: your_apple_id@example.com  # PUT YOUR APPLE ID HERE
-        password: Encrypted(...) # PUT YOUR APP-SPECIFIC-PASSWORD HERE https://support.apple.com/en-us/HT204397
+        api_key: $APP_STORE_CONNECT_PRIVATE_KEY      # Contents of the API key
+        key_id: $APP_STORE_CONNECT_KEY_IDENTIFIER    # Alphanumeric value that identifies the API key
+        issuer_id: $APP_STORE_CONNECT_ISSUER_ID      # Alphanumeric value that identifies who created the API key
+        submit_to_testflight: true        # Optional boolean, defaults to false. Whether or not to submit the uploaded build to TestFlight to automatically enroll your build to beta testers.
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md:144]:
```suggestion
        submit_to_testflight: false        # Optional boolean, defaults to false. Whether or not to submit the uploaded build to TestFlight to automatically enroll your build to beta testers.
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]
Set `submit_to_testflight: false` otherwise it will expect the beta_groups to be available.

**Review by nevercode-kevin — APPROVED** [review]
LGTM

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 5]
```diff
@@ -99,12 +99,14 @@ workflows:
   ios-workflow:
     name: ios_workflow
     environment:
+      groups:
+        - app_store_credentials # <-- Include - APP_STORE_CONNECT_ISSUER_ID, APP_STORE_CONNECT_KEY_IDENTIFIER, APP_STORE_CONNECT_PRIVATE_KEY
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md:103]:
```suggestion
        - app_store_credentials # <-- Includes: APP_STORE_CONNECT_ISSUER_ID, APP_STORE_CONNECT_KEY_IDENTIFIER, APP_STORE_CONNECT_PRIVATE_KEY
```

File: content/yaml-quick-start/building-a-native-ios-app.md [hunk 6]
```diff
@@ -99,12 +99,14 @@ workflows:
...
+        - certificate_credentials # <-- Include - CERTIFICATE_PRIVATE_KEY
```
Comment (nevercode-kevin) [inline content/yaml-quick-start/building-a-native-ios-app.md:104]:
```suggestion
        - certificate_credentials # <-- Includes: CERTIFICATE_PRIVATE_KEY
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]

**Review by nevercode-kevin — APPROVED** [review]
LGTM

**Pull request merged**